src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+		enterQuantity1.clear();
+		enterQuantity1.sendKeys("3");
+	}
+	
+	 @FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+	 //*[@id="change--6.quantityAsDecimal"]
+	  private WebElement enterQuantity2;
+
+	 		/**
+	 		 * Method to Enter Price Rate.
+	 		 * 
+	 		 * @throws IOException
+	 		 */
+	 public void enterQuantity2() throws IOException {
+		 
+		CreateOrderPage sp = new CreateOrderPage(driver);
+	 	log.info("Enter Quantity");
+	 	JavaScriptExec.sleep();
+	 	WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity2);
+	 	Assert.assertTrue(enterQuantity1.isDisplayed());
+	 	enterQuantity2.clear();
+	 	enterQuantity2.sendKeys(BasePage.getCellData(xlsxName, sheetName, 8, 2));
+	 		
+	 }
+	 
+	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+	 	//*[@id="change--6.quantityAsDecimal"]
+	 	private WebElement enterQuantity3;
+	 		/**
+	 		 * Method to Enter Price Rate.
+	 		 * 
+	 		 * @throws IOException
+	 		 */
+	 	public void enterQuantity3() throws IOException {
+	 		
+	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	log.info("Enter Quantity");
+	 	JavaScriptExec.sleep();
+	 	WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity2);
+	 	Assert.assertTrue(enterQuantity1.isDisplayed());
+	 	enterQuantity2.clear();
+	 	enterQuantity2.sendKeys(BasePage.getCellData(xlsxName, sheetName, 8, 3));
+	 		}
+	 	
+	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+ 	 	//*[@id="change--6.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity4;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity4() throws IOException {
+ 	 		
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity2);
+ 	 		Assert.assertTrue(enterQuantity1.isDisplayed());
+ 	 		enterQuantity4.clear();
+ 	 		enterQuantity4.sendKeys(BasePage.getCellData(xlsxName, sheetName, 8, 4));
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+ 	 	//*[@id="change--6.quantityAsDecimal"]
+ 	
+ 	 	private WebElement enterQuantity5;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity5() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity2);
+ 	 		Assert.assertTrue(enterQuantity1.isDisplayed());
+ 	 		enterQuantity5.clear();
+ 	 		enterQuantity5.sendKeys("2");
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--12.quantityAsDecimal']")
+ 	 	//*[@id="change--12.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity6;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
```

Comment:
@mayankchhajer Please remove the unused xpath

Reply:
Done